source/modulo_components/modulo_components/component_interface.py

```diff
@@ -1,7 +1,7 @@
 import inspect
 import sys
 from functools import partial
-from typing import Callable, Dict, List, Optional, TypeVar, Union
+from typing import Callable, Dict, List, Optional, TypeVar, Union, Iterable
```

Comment:
Why do you use Iterable and not List?

Reply:
Iterable is more generic than list. It can accept Tuple, Set and such. As long as the container can be iterated through.